aptos-move/script-composer/src/builder.rs

```diff
@@ -629,10 +636,12 @@ impl From<CallArgumentWasm> for CallArgument {
 
 #[wasm_bindgen(js_class = "CallArgument")]
 impl CallArgumentWasm {
+    #[wasm_bindgen(js_name = newBytes)]
```

Comment:
Just curious, what would happen if we don't have this? Is there a test for showing effects of this particular change?

Reply:
This is just for changing the wasm side names to align with our APIs in the ts-sdk. I will rebundle a newer version into our sdk to make sure this flow works.